src/components/Subscribe/_subscribe.scss

```diff
@@ -4,34 +4,57 @@
 @import '#scss/typography';
 
 .#{$px}-subscribe {
-  max-width: 30rem;
+  max-width: 600px;
   width: 100%;
 
+  .#{$px}-input {
+    &__label {
+      color: $pure-black;
+      font-size: 0.75rem;
+      text-transform: none;
```

Comment:
This should probably be applied to the input component. From that [figma](https://www.figma.com/design/hMu9IWH5N3KamJy8tLFdyV/EASEL-Compendium%3A-Tokens%2C-Components-%26-Patterns?node-id=7584-17668&m=dev) it looks like this applies to all inputs.

Reply:
Should the font size come from a text mixin?